dags/monrecap.py

```diff
@@ -41,11 +46,30 @@ def monrecap_airtable(**kwargs):
             df = airtable.fetch_airtable_data(url, headers)
 
             if table_name == "Commandes":
-                df["Submitted at"] = pd.to_datetime(df["Submitted at"])
+                # getting all the columns starting with date + Submitted at when needed.
+                # This is repeated for all imported tables
+                commandes_dates = [col for col in df.columns if date_pattern.match(col) or col == "Submitted at"]
```

Comment:
Tu aurais aussi pu faire comme ceci :
```suggestion
                commandes_dates = [col for col in df.columns if col.lower().startswith("date") or col == "Submitted at"]
```

Reply:
Oui j'y ai pensé ! mais je me suis dit que si Annaelle me demandait de gérer une nouvelle colonne qui ne commence pas par "date" je pouvais directement passer par le regex

Reply:
`.startwith()` accepte aussi un tuple ;) : https://docs.python.org/3/library/stdtypes.html#str.startswith